Parsers/ASimNetworkSession/Parsers/ASimNetworkSessionCheckPointFirewall.yaml

```diff
@@ -0,0 +1,260 @@
+  let ActionLookup=datatable(DvcOriginalAction:string,DvcAction:string)
+  [
+      "Accept","Allow", 
+      "Allow","Allow", 
+      "Drop","Drop", 
+      "Reject","Deny", 
+      "Encrypt","Encrypt", 
+      "Decrypt","Decrypt",
+      "Bypass","Allow", 
+      "Block","Deny", 
+      "Detect","Deny", 
+      "Do not send","Drop",  
+      "Authorize","", 
+      "Deauthorize","", 
+      "Ask User","", 
+      "Extract",""];
+let NWParser=(disabled:bool=false){
+    CommonSecurityLog
+    | where not(disabled)
+    | where DeviceVendor=="Check Point" and DeviceProduct=="VPN-1 & FireWall-1"
+    | extend 
+            EventProduct = "Firewall",
+            EventCount = toint(1),
+            EventType = "NetworkSession",
+            EventSchema = "NetworkSession",
+            EventSchemaVersion = "0.2.4"
+    | parse AdditionalExtensions with * "rule_action=" DeviceAction ";" * "rule_uid=" NetworkRuleName ";" *
+    | parse AdditionalExtensions with * "loguid=" EventOriginalUid ";" * "origin=" Dvc ";" * "originsicname=CN\\=" DvcHostname "," *
+    | parse AdditionalExtensions with * "inzone=" SrcZone ";" * "outzone=" DstZone ";" * 
+    | parse AdditionalExtensions with * "conn_direction=" CommunicationDirection ";" 
+    | parse AdditionalExtensions with * "alert=" AlertName ";" *
+    | parse AdditionalExtensions with * "inspection_category=" InspectionCategory ";" * "inspection_item=" ThreatName ";"                   
+    | extend
+            ThreatCategory = coalesce(AlertName, InspectionCategory),
+            EventResult = case(
+                                isempty(DeviceAction), "", 
+                                DeviceAction == "Accept", "Success",
+                                "Failure"
+                            ), 
+            NetworkRuleName = coalesce(DeviceCustomString2, NetworkRuleName),
+            EventStartTime = unixtime_milliseconds_todatetime(tolong(ReceiptTime))
+    | project-rename  
+            EventVendor = DeviceVendor,
+            DstPortNumber = DestinationPort,
+            DstIpAddr = DestinationIP,
+            SrcPortNumber = SourcePort,
+            SrcIpAddr = SourceIP,
+            DstNatIpAddr = DestinationTranslatedAddress,
+            DstNatPortNumber = DestinationTranslatedPort,
+            SrcNatIpAddr = SourceTranslatedAddress,
+            SrcNatPortNumber = SourceTranslatedPort,
+            EventProductVersion = DeviceVersion,
+            EventOriginalSeverity = LogSeverity,
+            Rule = NetworkRuleName,
+            DstAppName = Activity,
+            DvcOriginalAction = DeviceAction
+    | lookup ProtocolLookup on Protocol
+    | lookup DirectionLookup on CommunicationDirection
+    | lookup ActionLookup on DvcOriginalAction
+    | extend 
+            EventEndTime = EventStartTime,
+            IpAddr = SrcIpAddr,
+            Dst = DstIpAddr,
+            Src = SrcIpAddr,
+            EventMessage = Message,
+            EventSeverity = case(
```

Comment:
I would make the default "Informational". Also, the new DvcAction's probably need their own mapping tot "Low" or "Informational". Best to also add this to the lookup.

Reply:
Added in the same lookup.
Question: In case we have Threat details, should we not have high severity levels like "Medium", "High" and "Critical"?

Reply:
Make sense, let's do High. Did you actually see Threat details?

Reply:
I found one type of record where alert=spoofalert value was found.